init.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set +x
```

Comment:
Do we want `set -e` instead?

Reply:
For some reason I thought I had to disable commands that were being printed. I just removed it now.